app/config/settings_base.py

```diff
@@ -87,9 +88,25 @@
         'TEST': {
             'NAME': env.str('DB_NAME_TEST', 'test_service_control'),
         }
+    },
+    'bod': {
```

Comment:
@boecklic I guess this means migrating the BOD credentials to SSM soon 😀

Reply:
@schtibe yo, though since this can use the generic read-only `www-data` user it likely already exists